.github/workflows/python-ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, windows-2019]
-        python-version: [3.7, 3.8]
+        python-version: [3.8, 3.10.11]
```

Comment:
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```

Reply:
I will try it, but my previous attempt with python  3.10 failed (version not found). Do the quotation marks have any meaning? They didn't in the previous code.

Reply:
Ok, it is working now. Thanks.

Reply:
It removes some guess work as to what the type of the value is. Now it is string, as it should be (and not float, so `3.10` is verbatim and not converted to `3.1`).